tests/test_PsfexPsf.py

```diff
@@ -268,6 +270,12 @@ def testPsfexDeterminer(self):
         self.assertEqual(psf.computeBBox(), psf.computeKernelImage().getBBox())
         self.assertEqual(psf.computeBBox(), psf.getKernel().getBBox())
 
+    def test_dm_31466(self):
+        """Test that there is no seg-fault if order is set to 0."""
```

Comment:
I don't really object to this test really, but it doesn't actually test what the previous fail case was; setting this equal to zero never segfaultted at the config set point, the issue was that the config allowed the wrong value, and then segfaulted on running.

I think that this can be deleted, as this is really nothing more than a test that pexConfig options abide by their limits, and I'm sure that's tested extensively elsewhere.